src/clj/game/cards/resources.clj

```diff
@@ -1487,9 +1490,11 @@
 
    "Safety First"
    {:in-play [:hand-size {:mod -2}]
-    :events {:runner-turn-ends {:req (req (< (count (:hand runner)) (hand-size state :runner)))
-                                :msg (msg "draw a card")
-                                :effect (effect (draw 1))}}}
+    :events {:runner-turn-ends {:msg (msg "draw a card")
```

Comment:
Same as above, this will print even when no card was drawn.

Reply:
Bump

Reply:
Haha wups I had added the message to the effect, but forgotten to remove the original message.